README.md

```diff
@@ -609,8 +609,8 @@ To this end, we end up with a format of families that all adhere to the Zed data
 model but emulate the efficiency of Avro and Parquet.
 
 * ZSON is like JSON
-* ZNG is record-based like Avro
-* ZST is columnar like Parquet
+* [ZNG](https://github.com/brimdata/zed/blob/main/docs/formats/zng.md) is record-based like Avro
+* [ZST](https://github.com/brimdata/zed/blob/main/docs/formats/zst.md) is columnar like Parquet
```

Comment:
It's only after I added these hyperlinks that I saw you also hyperlink them further down in the doc when you summarize. However, I tend toward the "link it the first time you reference it" approach. I figure a little redundant linking is pretty harmless, so I left it in the spot further down as well.